src/gvmd.c

```diff
@@ -2717,7 +2736,13 @@ gvmd (int argc, char** argv, char *env[])
    * release gvm-checking, via option_lock. */
 
   if (osp_vt_update)
+#if OPENVASD
+    g_critical ("%s: Default openvasd VT update client set."
+                 " The --osp-vt-update command was not executed.",
+                 __func__);
```

Comment:
I would keep the message simple here as openvasd is enabled.
```suggestion
    g_critical ("%s: openvasd is enabled."
                 " The --osp-vt-update command was not executed.",
                 __func__);
```